src/scripts/components/widgets/NumPad.js

```diff
@@ -0,0 +1,120 @@
+import React, { Component } from 'react'
+import styled from 'styled-components'
+import Colors from 'components/foundations/base/Colors'
+
+type Props = {
+  onSetPin: () => String,
```

Comment:
Does this actually return a string? Doesn't look like the return value is used in this component.

Reply:
@bent0b0x i use this pass the pin to the parent component..

Reply:
On line 90 though the return value is not used